src/main/scala/com/example/CsvToJsonApp.scala

```diff
@@ -0,0 +1,47 @@
+package com.example
+
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.scaladsl.Behaviors
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Route
+
+import java.nio.file.Files
+import java.nio.file.Paths
+import scala.util.Failure
+import scala.util.Success
+
+object CsvToJsonApp {
+
+  private def startHttpServer(routes: Route)(implicit system: ActorSystem[_]): Unit = {
+    import system.executionContext
+    val serverHost = system.settings.config.getString("server.host")
+    val serverPort = system.settings.config.getInt("server.port")
+
+    val futureBinding = Http().newServerAt(serverHost, serverPort).bind(routes)
+    futureBinding.onComplete {
+      case Success(binding) =>
+        val address = binding.localAddress
+        system.log.info("Server online at http://{}:{}/", address.getHostString, address.getPort)
+      case Failure(ex) =>
+        system.log.error("Failed to bind HTTP endpoint, terminating system", ex)
+        system.terminate()
+    }
+  }
+
+  def main(args: Array[String]): Unit = {
+
+    // In case output folder does not exist.
+    Files.createDirectories(Paths.get("output"))
```

Comment:
Hardcoded while in other places that path is configurable.

Reply:
I had in mind to fix the configuration as mentioned in next steps. To be completely honest I added this part at the last minute when I was testing it and noticed that the folder may not exist.